src/cephadm/cephadm

```diff
@@ -6459,6 +6459,18 @@ def command_zap_osds(ctx: CephadmContext) -> None:
 ##################################
 
 
+def get_ceph_cluster_count(ctx):
+    # type: (CephadmContext) -> int
+    def is_valid_cluster_uuid(uuid_to_test):
+        # type: (str) -> bool
+        try:
+            uuid.UUID(str(uuid_to_test))
+            return True
+        except ValueError:
+            return False
+    return len([c for c in os.listdir(ctx.data_dir) if is_valid_cluster_uuid(c)])
```

Comment:
```suggestion
def get_ceph_cluster_count(ctx: CephadmContext) -> int:
    return len([c for c in os.listdir(ctx.data_dir) if is_fsid(c)])
```

Reply:
why not use the existing `is_fsid` func? likewise, let's try to use py3 style type hints where possible ..

Reply:
thanks @mgfritch, yeah it does the same, I didn't see that function :+1: